webpack.config.js

```diff
@@ -1,19 +1,28 @@
-'use strict';
 const path = require('path');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
 
 module.exports = {
-    mode: 'production',
```

Comment:
**Suggestion (non-blocking)** We should keep this. Or if we really want the dev. mode we can pass `development`. To patch this Build warning -- 

<img width="941" alt="image" src="https://github.com/gitpod-io/browser-extension/assets/55068936/eb5fd055-c7af-400a-8c4c-e1e5598137bc">